client/components/Comments/CommentsDumb.js

```diff
@@ -64,6 +69,8 @@ class Comments extends Component {
       this.setState({ initialLoad: false });
       this.scrollToNewestComment();
     }
+    else if (prevProps.comments.length !== this.props.comments.length)
```

Comment:
i don't think you want this. If I was scrolled up, reading some comments, I wouldn't want to be forced back to the bottom.

Reply:
You need to try it. 
This keeps the position of scroll for comments at the point it should be at, even when new comments are loaded. Without this it automatically scrolls the comments-box to the top when new comments are added.